curriculum/challenges/english/05-apis-and-microservices/mongodb-and-mongoose/install-and-set-up-mongoose.english.md

```diff
@@ -6,7 +6,7 @@ challengeType: 2
 
 ## Description
 <section id='description'>
-Add mongodb and mongoose to the project’s package.json. Then require mongoose. Store your mLab database URI in the private .env file as MONGO_URI. Connect to the database using mongoose.connect(<Your URI>)
+Add mongodb and mongoose to the project’s package.json. Then require mongoose. Store your mLab database URI in the private <code>.env</code> file as MONGO_URI. Connect to the database using <code>mongoose.connect(&lt;Your URI&gt;)</code>
```

Comment:
```suggestion
In this challenge you will install and set up Mongoose for your project
```

Reply:
I think this should be discussed further in a separate issue related to rewriting the instructions for this challenge.  Can we keep this PR focused on the original scope?

Reply:
Sure, great